README.md

```diff
@@ -1,19 +1,34 @@
 # sturdy-memory
 A simple inventory management system for the Mongoose system of SV Sticky. 
 
-Fallacious rooster has been created as per request of the 18'th Commissioner of Internal Affairs, in order to keep track of the "black box" the loss of products in the Mongoose sometimes can be.
+Fallacious rooster has been created as per request of the 18'th Commissioner of Internal Affairs, in order to keep track
+of the "black box" the loss of products in the Mongoose sometimes can be.
 
 ## Development 
 The guide below will tell you roughly how to get started with working on sturdy-memory.
 
 ## Koala
-Sturdy-memory uses Koala for comparing succesful transactions with the counted amount of the board member checking the amount.
+Sturdy-memory uses Koala for comparing successful transactions with the counted amount of the board member checking the
+amount.
 
 ## Server setup
-The server is written in.....? (To be determened)
+The server is written in.....? (To be determined)
+
+### Endpoints
```

Comment:
You might want to move this to a dedicated file or docs folder. `README.md` typically only contains a quickstart guide and general knowledge. Great that you added this though!